### ISSUE TITLE
openweathermap.org is not accurate

### ISSUE THREAD

**Issue by vporton** [jenslody/gnome-shell-extension-openweather#96]
It seems that temperatures shown by openweathermap.org are sometimes much lower than reality (so it was here in Ashkelon, Israel, at least sometimes).

I suggest to make forecast.io the default information provider.

**Comment by jenslody** [comment]
I will not do this, unless forecast.io can be used out of the box, either without api-key, or with an api-key for the extension (like openweathermap.org).
I will contact the forecast.io devs in the next days.
I will leave this open, so I don't forget it.

**Comment by peekaa** [comment]
Yes, the temperatures and other data from openweathermap.org are totally bad, here in center Europe. Forecast.io is good, but quite inconvenience with creating account. Another weather providers are in plan? Like yr.no?

**Comment by jenslody** [comment]
@peekaa:
see #134 and #70

**Comment by cdysthe** [comment]
I would like to add that Forecast.io, now Dark Sky has become far less accurate over the last year or so here in the US. We need Weather Underground which has been, and is, a rock solid service.

**Comment by russmac** [comment]
In South Australia, Openweathermap for me appears to take the temperature from the CBD despite it allowing local areas for selection.

Quite often states its 23c when accurate weather is available stating its 17c.

We have a a free weather API available from the government 'bureau of meteorology' with everything from forecasts to radar feeds however as per #135 it is XML and needs parsing.
